docs/release-notes.md

```diff
@@ -13,6 +13,7 @@ Major changes:
 Minor changes:
 
 - Restore formatting of progress reporting to pre 0.24.0 behavior.
+- Fix UUID mismatch bug using `blkid -p` instead of `lsblk` to bypass the cache
```

Comment:
`rdcore` changes usually go in "Internal changes" instead.

Also, how about:

> rootmap: Bypass cache when querying filesystem UUID for `root` karg

?